### PR TITLE
Add torch nansum

### LINKED ISSUE #4516: nansum

### PR DESCRIPTION
**Pull request by Dsantra92**
closes #4516

### CODE REVIEW

File: ivy/functional/frontends/torch/reduction_ops.py [hunk 1]
```diff
@@ -31,3 +31,10 @@ def any(input, dim=None, keepdim=False, *, out=None):
     if ivy.is_uint_dtype(input_dtype):
         ret = ivy.astype(ret, input_dtype, out=out)
     return ret
+
+
+def nansum(input, dim=None, keepdim=False, *, out=None):
```
Comment (juliagsy) [inline ivy/functional/frontends/torch/reduction_ops.py:36]: args should be identical to native: https://pytorch.org/docs/stable/generated/torch.nansum.html

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hey! Thanks for contributing! There are changes required and please request my review once again when you're done! Thanks!

File: ivy/functional/frontends/torch/reduction_ops.py (hunk 1 above)
Comment (Dsantra92) [reply ivy/functional/frontends/torch/reduction_ops.py:36]: `torch.nansum(input, *, dtype=None)` is just a shorter form of longer `torch.nansum(input, dim=None, keepdim=False, *, dtype=None)`.

Comment (juliagsy) [reply ivy/functional/frontends/torch/reduction_ops.py:36]: ahh sorry, I probably overlooked last time, my bad! Will re-review asap!

File: ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py [hunk 2]
```diff
@@ -281,6 +281,45 @@ def test_torch_any(
...
+@handle_cmd_line_args
+@given(
+    dtype_input_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        force_int_axis=True,
+        min_num_dims=1,
+        min_axis=-1,
+        max_axis=0,
+    ),
+    with_out=st.booleans(),
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py:293]: remove this as it's unused

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hey there! Great work so far! Apologies for the confusion in the last review. Just a small change needed before it's good to go! Please also double check the tests and ensure that the test for the function you're working on is passing! As usual, please request my review again when you're done! Thanks!

File: ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py (hunk 2 above)
Comment (Dsantra92) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py]: Forgot to remove it in the last commit.

**Comment by Dsantra92** [comment]
Not working on it rn, planning to complete it within few days.

**Comment by juliagsy** [comment]
Sure! Feel free to request a review (at the right pane) when you're ready! Thank you!

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**